LiteDB/Utils/Extensions/StreamExtensions.cs

```diff
@@ -20,5 +20,19 @@ public static void FlushToDisk(this Stream stream)
                 stream.Flush();
             }
         }
+
+        public static async Task FlushToDiskAsync(this Stream stream)
```

Comment:
why you created this extension method? Is doing nothing useful...

Reply:
Removed, thanks